sources/editor/Xenko.Core.Assets.Editor/View/DefaultPropertyTemplateProviders.xaml

```diff
@@ -684,9 +698,10 @@
                          DecimalPlaces="{Binding DecimalPlaces, Converter={xk:Chained {cvt:DifferentValuesToNull}, {xk:ToDouble}}, FallbackValue=6}"
                          WatermarkContentTemplate="{StaticResource DifferentValuesWatermarkTemplate}"
                          DisplayUpDownButtons="False">
-      <!--<i:Interaction.Behaviors>
-        <xk:NumericTextBoxTransactionalRepeatButtonsBehavior UndoRedoService="{Binding Session.UndoRedoService}"/>
-      </i:Interaction.Behaviors>-->
+      <i:Interaction.Behaviors>
+        <!--<xk:NumericTextBoxTransactionalRepeatButtonsBehavior UndoRedoService="{Binding Session.UndoRedoService}"/>-->
```

Comment:
Wouldn't that change make Undo/Redo not work?

Reply:
No, all those were already commented out (the entire `<i:Interaction.Behaviors>` was commented out), I've just left it in there but still kept the specific tag commented out.